plugins/database/src/low_level_odbc.cpp

```diff
@@ -738,7 +742,7 @@ cllExecSqlWithResultBV(
     const char *sql,
     std::vector< std::string > &bindVars ) {
 
-    rodsLog( LOG_DEBUG10, "%s", sql );
+    log_sql::debug("{}: {}", __func__, sql);
```

Comment:
Add more context to the messages so the reader understands what is being conveyed. For example: 
```
{}: SQL = [{}]
```